.github/actions/azure/k8s-versions.yaml

```diff
@@ -0,0 +1,10 @@
+# List of k8s version for AKS tests
+---
+include:
+  - version: "1.24"
```

Comment:
Are we unintentionally removing 1.23 test coverage here? The commit message says that this adds "all supported versions" so I'm not sure if the removal of 1.23 is intentional.

Reply:
1.23 is not supported on Azure anymore, so we removed it as part of our usual cycling (otherwise what happens the installation will fail consistently due the K8s version being refused by the Azure API on cluster creation) => https://learn.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli

Reply:
OK, thanks. Not clear from the commit message.

Reply:
This table was created from the supported k8s for AKS, https://learn.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#aks-kubernetes-release-calendar